src/main/java/com/icthh/xm/ms/entity/web/rest/XmEntityResource.java

```diff
@@ -338,13 +350,15 @@ public XmEntityResource(
     @GetMapping("/xm-entities/{idOrKey}/links/targets")
     @Timed
     @PostFilter("hasPermission({'returnObject': filterObject, 'log': false}, 'XMENTITY.LINK.TARGET.GET_LIST')")
+    @PrivilegeDescription("Privilege to get all the targets")
```

Comment:
Privilege to get all links with entities referenced by the entity

Reply:
suggest:
`Privilege to get all target links with entities referenced by indicated entity`